ui/page/start_page.go

```diff
@@ -178,14 +167,26 @@ func (sp *startPage) loadingSection(gtx layout.Context) layout.Dimensions {
 					})
 				}),
 				layout.Rigid(func(gtx layout.Context) layout.Dimensions {
-					return sp.networkType.Layout(gtx)
+					netType := sp.WL.Wallet.Net
+					if sp.WL.Wallet.Net == "testnet3" {
```

Comment:
`testnet3` this should be a constant as it's used in more than one place

Reply:
The display text should be `testnet not `testnet3` as `testnet3` is the name of the backend folder.

Reply:
I meant to say the if statement should check for something like `if sp.WL.Wallet.Net == Constants.TESNET3`